### PR TITLE
Add organization search prototype

### PR DESCRIPTION
**Pull request by l-vincent-l**
Partial solution for #33

### CODE REVIEW

File: client/javascripts/app.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import '../stylesheets/app.scss'
```
Comment (maukoquiroga) [inline client/javascripts/app.js]: Not needed, webpack takes care of

File: client/javascripts/search_organizations.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+require('./organization.tag')
+require('./search_organizations.tag')
+const riot = require('riot')
+riot.mount('search_organizations')
```
Comment (maukoquiroga) [inline client/javascripts/search_organizations.js]: ```riot.mount('./search_organizations')``` ?

File: client/webpack.config.js [hunk 3]
```diff
@@ -1,12 +1,24 @@
...
+            failOnHint: true
+        }
+    }
+})
 
 module.exports = {
-    entry: ['./javascripts/app.js', './stylesheets/app.scss'],
+    entry: {
+        app: './javascripts/app',
+        search_organizations: './javascripts/search_organizations.js'},
```
Comment (maukoquiroga) [inline client/webpack.config.js]: Didn't understand why we need to import css from app.js.

File: lib/transport_web/templates/layout/app.html.eex [hunk 4]
```diff
@@ -16,6 +16,5 @@
     <main id="main" role="main">
       <%= render(@view_module, @view_template, assigns) %>
     </main>
-    <script src="<%= static_path(@conn, "/js/app.js") %>"></script>
```
Comment (maukoquiroga) [inline lib/transport_web/templates/layout/app.html.eex]: That's needed there.

File: lib/transport_web/templates/page/search_organizations.html.eex [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<search_organizations></search_organizations>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/fetch/2.0.3/fetch.min.js"></script>
```
Comment (maukoquiroga) [inline lib/transport_web/templates/page/search_organizations.html.eex]: What is it for, fetch polyfill ?

Comment (l-vincent-l) [reply lib/transport_web/templates/page/search_organizations.html.eex]: yep

**Comment by maukoquiroga** [comment]
Rebase + Fixup to clean up a bit and GTM for me 👍

**Comment by l-vincent-l** [comment]
image
Seems good to merge to me

**Pull request merged**